### PR TITLE
[DX-487] Whitelist deps to include in the build process

### PR DESCRIPTION
**Pull request by nickbalestra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,19 @@ Externals are not bundled when packaging and publishing the component, for bette
  
 ### features
 - `Server Side Rendering` = server side rendering should work as for any other OpenComponent. Differently from the 'oc-template-react' template this template rely on [`renderToStaticMarkup()`](https://reactjs.org/docs/react-dom-server.html#rendertostaticmarkup) for SSR as no hidration is needed.
-- `css` is supported by converting any css imported into [Style Objects](https://www.w3schools.com/jsref/dom_obj_style.asp). This object is compatible with React Inline Styles. 
+- `css` is supported by converting any css imported into [Style Objects](https://www.w3schools.com/jsref/dom_obj_style.asp). This object is compatible with React Inline Styles.
+- `White list dependencies` to be inlcuded in the build process done by the compiler. To whitelist dependencies installed in the node_modules folder, add in the package.json of the component a `buildIncludes` list:
```
Comment (matteofigus) [inline README.md]: inlucuded typo, "Whitelist" without spaces between white and list

File: README.md [hunk 2]
```diff
@@ -42,7 +42,19 @@ Externals are not bundled when packaging and publishing the component, for bette
...
+  ```json
+    ...
+    oc : {
+      files: {
+        template: {
+          ...
+          buildIncludes: ['react-components-to-build']
```
Comment (matteofigus) [inline README.md]: can you json this (double quotes on all props) - also I would rename `react-components-to-build` to `react-component-to-build`

**Review by matteofigus — APPROVED** [review]
A couple of changes on the readme, little ones

**Review by matteofigus — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -42,7 +42,19 @@ Externals are not bundled when packaging and publishing the component, for bette
  
 ### features
 - `Server Side Rendering` = server side rendering should work as for any other OpenComponent. Differently from the 'oc-template-react' template this template rely on [`renderToStaticMarkup()`](https://reactjs.org/docs/react-dom-server.html#rendertostaticmarkup) for SSR as no hidration is needed.
-- `css` is supported by converting any css imported into [Style Objects](https://www.w3schools.com/jsref/dom_obj_style.asp). This object is compatible with React Inline Styles. 
+- `css` is supported by converting any css imported into [Style Objects](https://www.w3schools.com/jsref/dom_obj_style.asp). This object is compatible with React Inline Styles.
```
Comment (lastquestion) [inline README.md:45]: 👋 is this because emails require everything to be in inline styles?

Comment (matteofigus) [reply README.md:45]: Yes, this repo is still a WIP but it's a fully opinionated OC template to work with e-mails, therefore SSR with staticMarkup rendering, inlined styles (with possibility of style tag for media queries), etc. 👍 

The goal is to have all the transactional emails in react very soon and capable of fully integrating to https://github.com/opentable/design-tokens
Long term there may be opportunities to share React components, but this is not a priority on the short term.
What do you think?

**Pull request merged**